apps/src/applab/designElements/ImagePickerPropertyRow.jsx

```diff
@@ -1,4 +1,5 @@
-var showAssetManager = require('../../assetManagement/show.js');
+/* global dashboard */
```

Comment:
I think it's acceptable, but we should keep trying to let apps do a graceful fallback when dashboard is not available - for tests, and for the offline use-case that Phil is working on.